src/theme/index.ts

```diff
@@ -24,4 +24,7 @@ export const RootTheme = {
     radii: { none: '0', base: '4px', modal: '8px', circular: '1000px' },
     [BUTTON_KEY]: BUTTON_THEME,
     colors: ColorsTheme,
+    breakpoints: {
```

Comment:
Just adding `xs` to the breakpoints by default so that if someone extends our theme they don't have to do it themselves. The xs breakpoint is just the default behavior (no media query).

Reply:
👍